app/components/Header.tsx

```diff
@@ -63,6 +63,9 @@ export function Header({ email }: { email?: string }) {
               <NavLink className="usa-nav__link" to="/docs" key="/docs">
                 Documentation
               </NavLink>,
+              <NavLink className="usa-nav__link" to="/news" key="/news">
+                News
+              </NavLink>,
```

Comment:
We can't add this to the primary nav right now. It renders incorrectly when the user is logged in.

```suggestion
```